frontend/src/common/backend.ts

```diff
@@ -49,6 +50,12 @@ export type VaultDto = {
   authPrivateKey: string;
 };
 
+export type VaultUpdateDto = {
+  name?: string;
+  description?: string;
+  archived?: string 
+};
+
```

Comment:
See above: can't we reuse the VaultDto?

Reply:
removed in https://github.com/cryptomator/hub/pull/202/commits/861fc8b08fe23249fc2c8501c7b2c3a711365127